core/state_transition.go

```diff
@@ -311,15 +311,17 @@ func (st *StateTransition) TransitionDb() (*ExecutionResult, error) {
 		st.state.SetNonce(msg.From(), st.state.GetNonce(sender.Address())+1)
 		ret, st.gas, vmerr = st.evm.Call(sender, st.to(), st.data, st.gas, st.value)
 	}
-	if !st.evm.ChainConfig().IsLondon(st.evm.Context.BlockNumber) {
+	
+	isLondon := st.evm.ChainConfig().IsLondon(st.evm.Context.BlockNumber)
```

Comment:
We have `homstead` and `istanbul` pulled out above. To keep the naming style consistent, please use `london` instead of `isLondon`. (`isXXX` is not a canonical naming convention is Go AFAIK).